imap_processing/swe/tests/test_swe_science.py

```diff
@@ -20,20 +72,88 @@ def test_uncompress_algorithm():
     assert expected_value == returned_value
 
 
+def test_swe_raw_science_data(decom_test_data):
+    """This test and validate raw and derived data of SWE science data."""
+    # read validation data
+    test_data_path = f"{imap_module_directory}/swe/tests/science_packets"
+    raw_validation_data = pd.read_csv(
+        f"{test_data_path}/idle_export_raw.SWE_SCIENCE_20230927_172708.csv",
+        index_col="SHCOARSE",
+    )
+    data = decom_test_data[0]
+    validation_data = raw_validation_data.loc[data.data["SHCOARSE"].raw_value]
+
+    # should have two packets
+    assert len(decom_test_data) == 4
```

Comment:
```suggestion
    # should have four packets
    assert len(decom_test_data) == 4
```
or just delete since it is obvious from the code so you don't have to update it again later if this changes.

Reply:
make sense. removed it.